front/src/modules/simulationResult/components/SpaceTimeChartWrapper/WaypointsPanel.tsx

```diff
@@ -16,13 +16,15 @@ type WaypointsPanelProps = {
   setWaypointsPanelIsOpen: (open: boolean) => void;
   waypoints: PathOperationalPoint[];
   waypointsPanelData: WaypointsPanelData;
+  showOffsets?: boolean;
 };
 
 const WaypointsPanel = ({
   waypointsPanelIsOpen,
   setWaypointsPanelIsOpen,
   waypoints,
   waypointsPanelData: { filteredWaypoints, setFilteredWaypoints, projectionPath, timetableId },
+  showOffsets = true,
```

Comment:
nit: Same as before, would prefer like that

```suggestion
  hideOffsets = false,
```